### PR TITLE
Merge Quote methods into a single class instead of multiple versions

### PR DESCRIPTION
**Pull request by timrisi**

### CODE REVIEW

File: src/generator.csproj [hunk 1]
```diff
@@ -56,5 +56,8 @@
     <Compile Include="generator-enums.cs" />
     <Compile Include="generator-filters.cs" />
     <Compile Include="generator-typemanager.cs" />
+    <Compile Include="..\tools\common\Quote.cs">
```
Comment (rolfbjarne) [inline src/generator.csproj]: Shouldn't this be `StringUtils.cs`?

File: tools/common/StringUtils.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+
+namespace Xamarin.Utils {
+	public class StringUtils {
+
+		static char shellQuoteChar;
+		static char ShellQuoteChar {
```
Comment (spouliot) [inline tools/common/StringUtils.cs]: Sounds better / simpler to have this computed in the .cctor and avoid the call (if not inclined) and the check (twice in quote)

File: tools/mtouch/Application.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2013 Xamarin Inc. All rights reserved.
+﻿// Copyright 2013 Xamarin Inc. All rights reserved.
```
Comment (chamons) [inline tools/mtouch/Application.cs]: Unneeded diff?

**Comment by timrisi** [comment]
Just need the green check from @spouliot to merge

**Pull request merged**